components/camera/server.go

```diff
@@ -53,8 +53,12 @@ func (s *subtypeServer) GetImage(
 		return nil, err
 	}
 
-	if req.MimeType == "" {
-		req.MimeType = utils.MimeTypeRawRGBALazy
+	if req.MimeType == "" { // there is no explicitly requested mimetype
+		req.MimeType = utils.MimeTypeJPEG
+		props, _ := cam.Properties(ctx) //nolint:errcheck
+		if props.ImageType == DepthStream {
+			req.MimeType = utils.MimeTypePNG
```

Comment:
I don't like that this has to be called every time we call GetImage - it's a second gRPC request for every GetImage request. Is there a way to store the ImageType in the server struct, and then use the stored value? 

Potentially, create a field in the subtypeServer struct that stores the imageType, and maybe you do a call to getProperties ONCE, and store the found value within the struct, so that you don't have to do repeated callls to cam.Properties().

Reply:
Or rather, it might or might not be a gRPC request, it's not clear if underlying the call the Properties() would end up requesting gRPC. So we should still minimizes calls to Properties()

Reply:
Also, you need to fill the Properties struct that is returned by
`func (vs *videoSource) Properties(ctx context.Context) (Properties, error)`
in camera.go with the ImageType of the camera - right now you leave it empty.

Reply:
And check the error returned by Properties(), but just put a logger WARN message with the error

Reply:
Okok definitely a good call. This is done. Now we'll only do cam.Properties if the requested mimetype is nil **and** the server has no idea what the imageType is (and even then, store the value so we don't have to do it again).